<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionTemplateListDialog</class>
 <widget class="QDialog" name="ActionTemplateListDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список записей</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="pnlTree" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblPadding">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CTreeView" name="treeItems"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pnlList" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="CTableView" name="tblItems">
         <property name="whatsThis">
          <string comment="ура!">список записей</string>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCountRows">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QStatusBar" name="statusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>A status bar</string>
     </property>
     <property name="whatsThis">
      <string>A status bar.</string>
     </property>
     <property name="sizeGripEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblCreatePerson">
     <property name="text">
      <string>Автор записи</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="CDateEdit" name="edtBegCreateDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblCreateDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Дата создания с</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="lblForCreateDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="CDateEdit" name="edtEndCreateDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblActionType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Тип действия</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSpeciality">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="whatsThis">
        <string>выбрать текущую запись</string>
       </property>
       <property name="text">
        <string>Выбор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFilter">
       <property name="whatsThis">
        <string>изменить условие отбора записей для показа в списке</string>
       </property>
       <property name="text">
        <string>Фильтр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="whatsThis">
        <string>изменить текущую запись</string>
       </property>
       <property name="text">
        <string>Правка F4</string>
       </property>
       <property name="shortcut">
        <string>F4</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNew">
       <property name="whatsThis">
        <string>добавить новую запись</string>
       </property>
       <property name="text">
        <string>Вставка F9</string>
       </property>
       <property name="shortcut">
        <string>F9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="whatsThis">
        <string>выйти из списка без выбора</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbCreatePerson"/>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbSpeciality"/>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="CActionTypeComboBoxEx" name="cmbActionType"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSNILS">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>СНИЛС</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="edtSNILS">
     <property name="inputMask">
      <string>999-999-999 99; </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CTreeView</class>
   <extends>QTreeView</extends>
   <header>library/TreeView.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeItems</tabstop>
  <tabstop>tblItems</tabstop>
  <tabstop>cmbActionType</tabstop>
  <tabstop>cmbSpeciality</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbCreatePerson</tabstop>
  <tabstop>edtBegCreateDate</tabstop>
  <tabstop>edtEndCreateDate</tabstop>
  <tabstop>btnFind</tabstop>
  <tabstop>btnSelect</tabstop>
  <tabstop>btnFilter</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ActionTemplateListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
